client/src/components/admin/AdminNews.vue

```diff
@@ -1,6 +1,31 @@
 <template>
   <div id="adminNewsPanel" class="py-3">
-    <div class="table-responsive">
+    <div>
+      <h5>Newsletter</h5>
+      <div class="form-check m-1">
+        <input
+          id="flexCheckDefault"
+          v-model="includeAllUserEmails"
+          class="form-check-input"
+          type="checkbox"
+          @change="onIncludeAllChanged"
+        />
+        <label class="form-check-label" for="flexCheckDefault">
+          Sende Nachricht an alle Nutzer anstatt nur Newsletter Abonennten
+        </label>
+      </div>
+      <a
+        class="bi bi-envelope btn btn-outline-primary btn-sm"
+        :class="disableNewsLetter ? 'disabled' : ''"
+        :href="`mailto:?bcc=${userEmails.join(
+          ';'
```

Comment:
';' did not work for me. ',' does. However: [https://web.archive.org/web/20151116223616/http://www.sightspecific.com/~mosh/WWW_FAQ/multrec.html](url)

Reply:
Duh...

But "," works also for me... So I will change that